ui/partials/perf/graphsctrl.html

```diff
@@ -32,6 +32,11 @@
     <div id="data-display">
       <select ng-model="myTimerange" ng-options="timerange.text for timerange in timeranges track by timerange.value" ng-change="timeRangeChanged()" ng-disabled="seriesList.length == 0">
       </select>
+      <select ng-model="myMeasure" ng-change="myMeasureChanged()"  ng-init="myMeasure=mean">
+        <option value="mean" selected="selected">Mean</option>
```

Comment:
Hi Will @wlach ,
I would like to give this select a default value, so graph data will shows up after user select datarange. But it doesn't work no matter how to define a default key value for it. It still empty until user select a value for it.
![screenshot from 2015-05-28 13 45 42](https://cloud.githubusercontent.com/assets/2166227/7853441/d28aad0a-0540-11e5-89b2-1bbb502eb163.png)